lang/en.json

```
@@ -338,6 +338,15 @@
     "TraitSelectorTitle": "Trait Selection",
     "TraitSelectorUpdate": "Update",
     "TransformerEnergon": "Energon",
+    "Upgrade": "Upgrade",
+    "UpgradeArmor": "Armor Upgrade",
+    "UpgradeAvailability": "Availability",
+    "UpgradeBenefit": "Benefit",
+    "UpgradeDrone": "Drone Upgrade",
+    "UpgradePrerequisite": "Prerequisite",
+    "UpgradeType": "Type",
+    "UpgradeWeapon": "Weapon Upgrade",
+    "Upgrades": "Upgrades",
```

```suggestion
    "Upgrade": "Upgrade",
    "UpgradeAvailability": "Availability",
    "UpgradeBenefit": "Benefit",
    "UpgradePlural": "Upgrades",
    "UpgradePrerequisite": "Prerequisite",
    "UpgradeTypeArmor": "Armor",
    "UpgradeTypeDrone": "Drone",
    "UpgradeTypeWeapon": "Weapon",
    "UpgradeType": "Type",
```
Having "Blah Upgrade" for all the types seems redundant to me. I changed `Upgrades` to `UpgradePlural` as well.